extensions/stats/plugin.cc

```diff
@@ -190,60 +186,89 @@ const std::vector<MetricTag>& PluginRootContext::defaultTags() {
 const std::vector<MetricFactory>& PluginRootContext::defaultMetrics() {
   static const std::vector<MetricFactory> default_metrics = {
       // HTTP, HTTP/2, and GRPC metrics
-      MetricFactory{
-          "requests_total", MetricType::Counter,
-          [](const ::Wasm::Common::RequestInfo&) -> uint64_t { return 1; },
-          false, count_standard_labels},
-      MetricFactory{
-          "request_duration_milliseconds", MetricType::Histogram,
-          [](const ::Wasm::Common::RequestInfo& request_info) -> uint64_t {
-            return request_info.duration /* in nanoseconds */ / 1000000;
-          },
-          false, count_standard_labels},
+      MetricFactory{"requests_total", MetricType::Counter,
+                    [](::Wasm::Common::RequestInfo&) -> uint64_t { return 1; },
+                    static_cast<uint32_t>(Protocol::HTTP) |
+                        static_cast<uint32_t>(Protocol::GRPC),
+                    count_standard_labels, false},
```

Comment:
nit: worth adding `/* not periodic */` or whatever for the `false` and similar for the `true` values in these definitions?